client/src/components/TreeImageScrubber.js

```diff
@@ -654,34 +606,84 @@ function SidePanel(props){
             />
           </Tabs>
           {switchApprove === 0 &&
-            <RadioGroup
-              value={captureApprovalTag}
-            >
-              <FormControlLabel 
-                onClick={() => handleCaptureApprovalTag('simple_leaf')}
-                value='simple_leaf' control={<Radio/>} label='Simple leaf' />
-              <FormControlLabel 
-                onClick={() => handleCaptureApprovalTag('complex_leaf')}
-                value='complex_leaf' control={<Radio/>} label='Complex leaf' />
-              <FormControlLabel 
-                onClick={() => handleCaptureApprovalTag('acacia_like')}
-                value='acacia_like' control={<Radio/>} label='Acacia-like' />
-              <FormControlLabel 
-                onClick={() => handleCaptureApprovalTag('conifer')}
-                value='conifer' control={<Radio/>} label='Conifer' />
-              <FormControlLabel 
-                onClick={() => handleCaptureApprovalTag('fruit')}
-                value='fruit' control={<Radio/>} label='Fruit' />
-              <FormControlLabel 
-                onClick={() => handleCaptureApprovalTag('mangrove')}
-                value='mangrove' control={<Radio/>} label='Mangrove' />
-              <FormControlLabel 
-                onClick={() => handleCaptureApprovalTag('palm')}
-                value='palm' control={<Radio/>} label='Palm' />
-              <FormControlLabel 
-                onClick={() => handleCaptureApprovalTag('timber')}
-                value='timber' control={<Radio/>} label='Timber' />
-            </RadioGroup>
+            <Grid container direction={'column'} className={classes.sidePanelContainer}>
+              <Grid>
+                <Typography variant='h6' >Tags</Typography>
```

Comment:
I think this title should be above the tabs as it applies to both approval and rejection

Reply:
Scrap that - it's fine as it is.